modules/dwh-networking/versions.tf

```
@@ -16,4 +16,34 @@
 
 terraform {
   required_version = ">= 0.13"
+
+
+  required_providers {
+
+    google = {
+      source  = "hashicorp/google"
+      version = "~> 3.77"
+    }
+
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = "~> 3.77"
+    }
+
+    null = {
+      source  = "hashicorp/null"
+      version = "3.2.0"
+    }
+
+    time = {
+      source  = "hashicorp/time"
+      version = "0.9.1"
```

Avoid breaking change

```suggestion
      version = ">= 0.7"
```